Tests/Realm.Tests/Database/NotificationTests.cs

```diff
@@ -57,14 +57,11 @@ public void ShouldTriggerRealmChangedEvent()
         [Test]
         public void RealmError_WhenNoSubscribers_OutputsMessageInConsole()
         {
-            using var sw = new StringWriter();
-            var original = Logger.Default;
-
-            Logger.Default = Logger.Function(sw.WriteLine);
+            var logger = new Logger.InMemoryLogger();
+            Logger.Default = logger;
```

Comment:
Do we need to reset the original logger?

Reply:
This is done automatically in [`RealmTest.TearDown`](https://github.com/realm/realm-dotnet/blob/c05396649345c644b0eb5786f8d2a42f117a05c9/Tests/Realm.Tests/RealmTest.cs#L86-L87)